plexpy/api2.py

```diff
@@ -524,13 +506,13 @@ def _api_run(self, *args, **kwargs):
 
             # We allow this to fail so we get a
             # traceback in the browser
-            if self._api_debug:
+            try:
                 result = call(**self._api_kwargs)
-            else:
-                try:
-                    result = call(**self._api_kwargs)
-                except Exception as e:
-                    logger.error(u'PlexPy APIv2 :: Failed to run %s %s %s' % (self._api_cmd, self._api_kwargs, e))
+            except Exception as e:
+                if self._api_debug:  # check this with j
```

Comment:
@JonnyWong16 Whats your take this on? I would like to get the traceback in the browser if the debug param is used with the api.